docs/diagrams/container.puml

```diff
@@ -0,0 +1,40 @@
+@startuml container
+
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Container.puml
+
+skinparam linetype polyline
+
+title [Container] HMPPS Integration API
+
+AddElementTag("not_yet_implemented", $bgColor="#D3D3D3", $fontColor="#9C9C9C")
+
+System_Ext(consumer, "Consumer Application", "An authenticated consumer of the Integration API.")
+
+Boundary(aws_cloudplatform, "Cloud Platform", "A Kubernetes cluster using Amazon Web Services", $link="https://user-guide.cloud-platform.service.justice.gov.uk/documentation/concepts/cp-tech-overview.html") {
+    Boundary(hmpps_integration_api, "HMPPS Integration API", "A single point of entry for services to retrieve data from multiple HMPPS systems."){
+        Container(springboot_api, "REST API", "Springboot Kotlin API", "Uses gateways to pull data from upstream systems. DTO's are configured to transform the format of the data as it passes through.", $link="https://github.com/ministryofjustice/hmpps-integration-api")
```

Comment:
Spring Boot is two words! 😝 

![image](https://user-images.githubusercontent.com/42817036/220064745-61d9ee33-e8e7-4980-8322-8cffa03eb3d8.png)

Reply:
```suggestion
        Container(springboot_api, "REST API", "Springboot Kotlin API", "Pulls data from upstream systems and transforms the format of the data as it passes through.", $link="https://github.com/ministryofjustice/hmpps-integration-api")
```

I think mentioning gateways and DTOs are implementation details and the important thing to explain is the "what" and less so the "how".